wallet/lib/pages/purchase_item/widgets/pay_now_dialog.dart

```diff
@@ -248,81 +214,99 @@ class _PayNowWidgetState extends State<PayNowWidget> {
-                title,
-                style: _rowTitleTextStyle,
-              ),
-              SizedBox(
-                width: 5.w,
-              ),
-              InkWell(
-                  onTap: () {
-                    ScaffoldMessenger.of(context)
-                        .showSnackBar(SnackBar(
-                          content: Text(LocaleKeys.pylons_fee_msg.tr()),
-                          margin: const EdgeInsets.all(10),
-                          behavior: SnackBarBehavior.floating,
-                        ))
-                        .closed
-                        .then((value) => ScaffoldMessenger.of(context).clearSnackBars());
-                  },
-                  child: SvgPicture.asset(
-                    SVGUtil.i_icon,
-                    color: AppColors.kWhite,
-                    fit: BoxFit.cover,
-                  )),
-            ],
-          ),
-        )),
-        Expanded(
+          flex: 2,
           child: Text(
             subtitle,
-            style: _rowSubtitleTextStyle,
+            maxLines: 1,
+            overflow: TextOverflow.ellipsis,
+            style: TextStyle(color: subtitleTextColor, fontSize: 15.sp, fontWeight: FontWeight.w600),
           ),
-        )
+        ),
       ],
     );
   }
 
   Widget buildButton({required String title, required Color bgColor, required Function onPressed}) {
     return SizedBox(
       height: 40.h,
-      child: InkWell(
-        onTap: () => onPressed(),
-        child: CustomPaint(
-          painter: BoxShadowPainter(cuttingHeight: 18.h),
-          child: ClipPath(
-            clipper: MnemonicClipper(cuttingHeight: 18.h),
+      child: Padding(
+        padding: EdgeInsets.symmetric(horizontal: 20.w),
+        child: InkWell(
+          onTap: () => onPressed(),
+          child: CustomPaint(
+            painter: BoxShadowPainter(cuttingHeight: 18.h),
+            child: ClipPath(
+              clipper: MnemonicClipper(cuttingHeight: 18.h),
+              child: Container(
+                color: bgColor,
+                height: 40.h,
+                child: Center(
+                  child: Text(
+                    title,
+                    style: TextStyle(color: AppColors.kWhite, fontSize: isTablet ? 14.sp : 16.sp, fontWeight: FontWeight.w700),
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildPriceView() {
+    return Padding(
+      padding: EdgeInsets.symmetric(horizontal: 20.w),
+      child: Row(
+        children: [
+          Expanded(
+            flex: 2,
+            child: Container(
+              height: 40.h,
+              color: AppColors.kDarkRed,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  SizedBox(width: 12.w, height: 12.w, child: widget.nft.ibcCoins.getSecondaryAssets()),
```

Comment:
Should height here be .h instead of .w?

Reply:
@samarali92  @Amean The original idea was to have same height and width. Thats why we used .w for both height and width. But thats obviously doesn't make sense. So instead of `.w` we can use `.r`for both height and width.